### ISSUE TITLE
Hooking in to the primary clipboard deselects all Ctrl-A + complete desktop freezes

### ISSUE THREAD

**Issue by jtagcat** [Linus789/wl-clip-persist#11]
`wl-clip persist -c regular` works fine. With `-c primary` or `-c both`, the following abnormalities happen:

<details>
<summary>Dupe of #3</summary>
### Ctrl-A
Selecting text (most commonly with Ctrl-A) in dialogues is consistently buggy. Anything selected is instantly deselected.

Examples: Thunar (rename), GIMP (Ctrl-N pixels).
Possibly related: https://github.com/Linus789/wl-clip-persist/issues/6
</details>

### Complete desktop freezes
Using Sway (1.9-3) on Arch Linux. A well-reproducible example with GIMP (2.10.38-1):
1. Ctrl-N: Create a new document
2. Ctrl-S: Attempt to open the dialogue to Save (As). If it doesn't crash on first time, try again. I can reproduce in 1-3 times looping Ctrl-S Esc.
3. Sway / the system is frozen. Time stops ticking. Switching TTY with Ctrl-Alt-F4 doesn't work. Power/Sleep inputs are ignored. Have to force shut-down.

If desired, I can reproduce and collect logs with SSH.

Any drop-downs from the top (File, Edit, Select) don't open, open very slowly, or cause the same freeze.

**Comment by Linus789** [comment]
The first issue is the same as #3. I can reproduce the second issue (GIMP crashing, but not the system freezing), but I really don't think it's wl-clip-persist's fault, since it worked before.

TL;DR: Probably not a bug from wl-clip-persist, just use regular mode.

**Comment by jtagcat** [comment]
Should I report it to [GIMP](https://gitlab.gnome.org/GNOME/gimp/-/issues)?

**Comment by Linus789** [comment]
I can’t reproduce the GIMP crash anymore with GIMP 3.0.2.
